Doc/library/socket.rst

```diff
@@ -363,6 +378,16 @@ Constants
    .. versionadded:: 3.7
 
 
+.. data:: AF_PACKET
+          PF_PACKET
+          PACKET_*
+
+   Many constants of these forms, documented in the Linux documentation, are
```

Comment:
"Linux documentation" -> "man pages"

Reply:
"Linux documentation" is used in the other definitions within this section, so I had used it for consistency.  Should I change them all?

Reply:
Ah, sorry, I didn't realize that term was already in use here. I suppose this is fine to leave as is.

Reply:
I think it would make sense to change that, but that should probably be another change.